### PR TITLE
[Enhancement] (nereids)implement helpCommand in nereids

### LINKED ISSUE #42824: [Enhancement] (nereids)implement helpCommand in nereids
help command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following help command in nereids:
unsupportedhelpStatement
``` HELP mark=identifierOrText                                                    ```

### PR DESCRIPTION
**Pull request by msridhar78**
### What problem does this PR solve?

Issue Number: close #42824 

Related PR: #xxx

Problem Summary:

[Enhancement] (nereids)implement helpCommand in nereids

### Release note

None

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by msridhar78** [comment]
run buildall

**Comment by LiBinfeng-01** [comment]
pls resolved conflicts

**Comment by msridhar78** [comment]
run buildall

**Comment by msridhar78** [comment]
run buildall

**Comment by msridhar78** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/HelpCommand.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        }
+        return resultSet;
+    }
+
+    @Override
+    public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
+        return visitor.visitHelpCommand(this, context);
+    }
+
+    public ShowResultSetMetaData getMetaData() {
```
[the remark this answers is not in the document]
Comment (msridhar78) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/HelpCommand.java]: done - pls review

**Comment by msridhar78** [comment]
run buildall

**Comment by msridhar78** [comment]
run buildall

**Comment by msridhar78** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by LiBinfeng-01 — APPROVED** [review]

**Pull request merged**